packages/generators/README.md

```diff
@@ -0,0 +1,9 @@
+# Generator
+
+Generate a sub-package under wp-calypso
+
+## Usage
+
+```
+yarn run generate
```

Comment:
Maybe it's worth covering the different types of templates and providing some examples and usage. Using existing packages will suffice IMHO.

Reply:
Updated!